src/extensions/averagewaveforms2/AverageWaveformsView/fetch_average_waveforms_2.py

```diff
@@ -0,0 +1,54 @@
+from typing import Dict
```

Comment:
add hither function to retrieve waveforms. This was mostly copied from existing function